src/Altinn.App.Core/Extensions/CollectionExtensions.cs

```diff
@@ -0,0 +1,23 @@
+namespace Altinn.App.Core.Extensions;
+
+/// <summary>
+/// Extensions for Collections.
+/// </summary>
+public static class CollectionExtensions
```

Comment:
Don't think this should be public

Reply:
I think we should add `NotNullWhenAttribute` to let the compiler know that `enumerable` is not null when the method returns false

Reply:
resolved as the class will be removed.